docs/user/index.rst

```diff
@@ -90,14 +90,13 @@ to help you create fantastic documentation for your project.
 
 .. toctree::
    :maxdepth: 2
-   :hidden:
```

Comment:
Seems this is showing on the index page 2x now?

Reply:
Not for me, or maybe there's some way I didn't get this. Anyway, I've re-added it now. I don't quite grasp from the documentation why it's needed... but since it's in so many other TOCs, we can tackle it when the time feels right :)

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-toctree

Reply:
@benjaoming It makes it so the TOCTree doesn't show up in the page where it's defined. We're managing the TOC's manually in RST, so if we remove this, it will output _another_ TOC. If you remove it and rebuild the index, you can see it.